task-buffer/task-buffer.ts

```diff
@@ -11,7 +11,7 @@ import {
   spawn,
   type Task,
   useScope,
-} from "effection";
+} from "npm:effection@3.0.3";
```

Comment:
Same as above. We updated the import map, but then we aren't using it.

Reply:
I added the import map because scopes can only be defined in root - it was needed to specify effection dependency for the files imported from the effection website. I'm thinking that I should publish those dependencies from the effection website to JSR instead. What do you think?